op-mode-definitions/container.xml.in

```diff
@@ -103,27 +103,54 @@
         </properties>
         <command>sudo ${vyos_op_scripts_dir}/container.py show_container</command>
         <children>
-          <leafNode name="image">
+          <node name="json">
+            <properties>
+              <help>Show containers in JSON format</help>
+            </properties>
+            <!-- no admin check -->
```

Comment:
What does "no admin check" mean?

Reply:
I assume it means it can be run with sudo without a password prompt? Those comments were just copied from the other nodes in this file to keep it consistent, I assumed it had some meaning.